src/components/Comment/CommentReply.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <div class="comment-reply-container">
+    <div class="comment-by">
+      {{ reply.createdBy }} - {{ reply.createdTime | moment("timezone", "America/Toronto", "DD MMM YYYY HH:MM") }}
```

Comment:
can we use moment timezone's feature to guess user timezone instead of hard coded timezone?